synapse/handlers/space_summary.py

```diff
@@ -200,6 +254,43 @@ async def _summarize_local_room(
             )
         return (room_entry,), events_result
 
+    async def _summarize_remote_room(
+        self,
+        room: "_RoomQueueEntry",
+        suggested_only: bool,
+        max_children: Optional[int],
+        exclude_rooms: Iterable[str],
+    ) -> Tuple[Sequence[JsonDict], Sequence[JsonDict]]:
+        room_id = room.room_id
+        logger.info("Requesting summary for %s via %s", room_id, room.via)
+
+        # we need to make the exclusion list json-serialisable
+        exclude_rooms = list(exclude_rooms)
+
+        via = itertools.islice(room.via, MAX_SERVERS_PER_SPACE)
+        try:
+            res = await self._federation_client.get_space_summary(
+                via,
+                room.room_id,
+                suggested_only=suggested_only,
+                max_rooms_per_space=max_children,
+                exclude_rooms=exclude_rooms,
+            )
+        except Exception as e:
+            logger.warning(
+                "Unable to get summary of %s via federation: %s",
+                room_id,
+                e,
+                exc_info=logger.isEnabledFor(logging.DEBUG),
```

Comment:
Why only send this if the debug logging is enabled?

Reply:
basically because in 90% of cases, it's going to be overly verbose: it'll be due to servers not supporting the endpoint, or having left the room, or being down, or whatever - none of which merit a stacktrace. On the other hand, if it's going wrong for reasons we don't understand, it'll be useful to be able to see the stacktrace.